### PR TITLE
Adjust chevron positioning

### PR DESCRIPTION
**Pull request by dkasper-was-taken**
### Summary
Correct chevron positioning with new arrange changes.

Thanks for contributing to Terra. 
@cerner/terra

### CODE REVIEW

File: packages/terra-list/src/List.scss [hunk 1]
```diff
@@ -27,11 +27,14 @@
   // ========================================================
 
   .chevron {
+    display: flex;
     padding-left: 10px;
     padding-right: 10px;
 
     > svg {
```
Comment (emilyrohrbough) [inline packages/terra-list/src/List.scss]: previously would this scale with font-size?

File: packages/terra-list/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
+* Chevron positioning
```
Comment (bjankord) [inline packages/terra-list/CHANGELOG.md]: More details on this fix would be helpful if we every need to modify this in the future. Like what browsers it was an issue in, etc.

Comment (kschuste) [reply packages/terra-list/CHANGELOG.md]: could include that you applied it's using flexbox now as well

**Comment by mmalaker** [comment]
Need more detail to functionally verify. Where was the positioning problem occuring? What components are affected?

**Comment by StephenEsser** [comment]
@mmalaker 

There may be more areas to verify, the one area I know of that was an issue was the Single Select Lists chevron position that can be seen on the doc site. 

Before Change:
http://engineering.cerner.com/terra-core/#/site/list/single-select

After Change:
https://terra-core-deployed-pr-988.herokuapp.com/static/#/site/list/single-select

**Comment by mmalaker** [comment]
Functional verification completed. The examples with chevron icons now display vertically centered.

**Comment by bjankord** [comment]
This has been released in terra-list 1.16.0

**Pull request merged**